### ISSUE TITLE
[7.7.0] `getSourceModuleName` should only be called on `MemberExpression` or `Identifier`

### ISSUE THREAD

**Issue by boris-petrov** [ember-cli/eslint-plugin-ember#614]
After updating, I get that error:

```
Build Error (broccoli-persistent-filter:EslintValidationFilter) in components/application-header.ts

`getSourceModuleName` should only be called on `MemberExpression` or `Identifier`
Occurred while linting /home/boris/project/app/components/application-header.ts:24
```

That line is as follows:

```typescript
@classic
@tagName('nav')
@classNames('navbar', 'navbar-default', 'navbar-fixed-top')
export default class extends Component.extend(ResizeAware) {
```

The last line is `24`.

**Comment by bmish** [comment]
Thanks for the report. I will fix this within a few hours.
